ErsatzTV/client-app/src/views/WatermarksPage.vue

```diff
@@ -0,0 +1,97 @@
+﻿<template>
```

Comment:
Did you intend to include this file as part of this FFmpegProfile PR?

Reply:
I started this before realizing the new ffmpeg files weren't in, this came along with that push. Will remove.